src/core/probe/manager.rs

```diff
@@ -249,9 +229,13 @@ impl ProbeManager {
 
         // First, handle generic probes.
         let mut set = ProbeSet {
-            probes: self.generic_probes.clone(),
+            probes: self
+                .probes
+                .clone()
```

Comment:
Not sure if this comment will be fixed later on in the same series but IIUC the ProbeSet is now a temporary object so we could give it a reference to probes and avoid cloning.